docs/changelog/116731.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 116731
```

Comment:
We don't need both changelogs. This new one also has a less correct summary, so rather keep the old one. Since github actions will regenerate this, we can skip that by adding the `backport` label.

Reply:
Dםמקץ